src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/src/extension.ts

```diff
@@ -23,12 +23,13 @@ export async function activate(context: vscode.ExtensionContext) {
         __dirname, '..', '..', '..', '..', '..', 'artifacts', 'bin', 'rzls', config, 'netcoreapp5.0');
 
     if (!fs.existsSync(languageServerDir)) {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Instead of suppressing this every time there's a promise is there a better way to have "fire and forget" promises?